app/views/contestants/index.html.erb

```diff
@@ -0,0 +1,10 @@
+<h1>Contestants</h1>
+<% @contestants.each do |contestant| %>
+  <h3><%= contestant.name %></h3>
+  <p>
+    Projects:
+    <% contestant.projects.each_with_index do |project, index| %>
+      <%= project.name %><%= ',' unless index == contestant.projects.length - 1 %>
```

Comment:
`contestant.projects.join(", ")` is perhaps an easier way to achieve this same formatting